src/css/style.scss

```diff
@@ -1,17 +1,28 @@
+$background-color: #4d4e4f;
+$h1-font-color: #ffffff;
+$body-font-family: 'Open Sans';
+$sidebar-color: #9A9DAB;
+
```

Comment:
?? Anyone know if we should have a line space between different types of variables, or do they all need to be all grouped together?